src/extension.ts

```diff
@@ -1048,37 +1049,40 @@ async function selectPod(scope: PodSelectionScope, fallback: PodSelectionFallbac
     return value.pod;
 }
 
-async function logsKubernetes(explorerNode?: explorer.ResourceNode) {
+async function logsKubernetes(explorerNode?: explorer.ResourceNode, follow?: boolean) {
     if (explorerNode) {
         const podSummary = { name: explorerNode.id, namespace: undefined };  // TODO: namespaces
         const container = await selectContainerForPod(podSummary);
         if (container) {
-            getLogsForContainer(podSummary.name, podSummary.namespace, container.name);
+            getLogsForContainer(podSummary.name, podSummary.namespace, container.name, follow);
         }
     } else {
-        findPod(getLogsForPod);
+        findPod((pod: PodSummary) => {getLogsForPod(pod, follow);});
```

Comment:
I don't know if we need the type declaration here - think TypeScript should infer it from `findPod` and `getLogsForPod`.  But if not then keep it.